src/resolvers/claims.ts

```diff
@@ -30,20 +41,34 @@ export const claims: QueryResolvers['claims'] = async (
   return claims.map((c) => transformClaim(c, strings))
 }
 
+
+const signURL = (bucket: String, key: String) => s3.getSignedUrl('getObject', {
+  Bucket: bucket,
+  Key: key,
+  Expires: THIRTY_MINUTES,
+})
```

Comment:
Might be wise to use `getSignedUrlPromise` (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getSignedUrlPromise-property)

```suggestion
const signURL = (bucket: String, key: String) => s3.getSignedUrlPromise('getObject', {
  Bucket: bucket,
  Key: key,
  Expires: THIRTY_MINUTES,
})
```

This change would change the signature of this from `string` to `Promise<string>`, which means that callers would need to `await` the call to `signURL`.

Reply:
getSignedUrlPromise does not seem exist in the version of the aws-sdk we are using. So I'll skip that for now? getSignedUrl is being used in more places as well.

Reply:
I see! We can trivially create our own by wrapping it:

```typescript
const getSignedUrl = new Promise((resolve, reject) => {
  s3.getSignedUrl(
    'getObject',
	{
	  Bucket: bucket,
      Key: key,
  	  Expires: THIRTY_MINUTES,
  	},
    (err, url) => {
      if (err) {
		reject(err)
        return
	  }
      resolve(url)
    }
  )
})
```